variables.tf

```diff
@@ -83,6 +83,12 @@ variable "container_port" {
   type        = "string"
 }
 
+variable "nlb_healthcheck_port" {
```

Comment:
I'm not sure if these needs to be specific the NLB. It could just be `container_health_check_port`. That way you could extend this functionality if it's required for the ALB.

Reply:
Did you want to add functionality for the ALB use cases?  (even though we won't be using it right now) . Or just rename the variable?

Reply:
My motivation is trying to make this module generally useful since it's public. So yeah I think adding the ALB case would round out this PR nicely.